pipelines/broad/dna_seq/germline/joint_genotyping/UltimaGenomics/UltimaGenomicsJointGenotyping.changelog.md

```diff
@@ -0,0 +1,4 @@
+# 1.0.0
+20210-05-09 (Date of Last Commit)
+
```

Comment:
@meganshand what do we wanna write for this changelog entry? also small typo in the year

Reply:
Fixed and added an entry.